docs/drill4net/tests-project-setup.mdx

```diff
@@ -24,6 +24,16 @@ var driver = new ChromeDriver().UseDrill4Net();
 ```
 Make sure you added `using Drill4Net.Selenium.Drill4NetDriver;`.
 
+## Playwright Tests Setup
```

Comment:
Add Playwright Tests to the **Supported Tests** section (line 8)

Reply:
added